setup.py

```diff
@@ -30,12 +34,16 @@ def get_version():
     'pytest-asyncio>=0.5.0',
     'pytest-cov>=2.3.1',
     'logbook>=1.0.0',
+    'flake8>=3.0.4',
+    'isort>=4.2.5',
+    'collective.checkdocs>=0.2',
+    'Pygments>=2.1.3'  # required by checkdocs
```

Comment:
But running Travis will only allow allow you to fix the problem in a new release. On the other hand, it will break all existing deployment setups out there, if the package is installed on the server via pip.

User experience: Installing saltyrtc-server==1.0.0 on the server worked yesterday, but installing the same version on another server today with exactly the same setup results in strange exceptions.